merch-dropper/src/utils/axiosWithAuth.js

```diff
@@ -11,7 +11,8 @@ export const axiosWithAuth = () => {
   const token = localStorage.getItem("token");
 
   return axios.create({
-    baseURL: "https://merchdropper-production.herokuapp.com",
```

Comment:
look here

Reply:
We want it to go to merch-dropper.herokuapp.com now. Do I have that right?